src/BUILD

```diff
@@ -469,6 +477,24 @@ cc_library(
     alwayslink = 1,
 )
 
+pybind_extension(
+    name = "pyovms",
+    srcs = ["bindings/python/ovms_py_tensor.cpp",
+            "ovms_py_tensor.hpp",
+            "ovms_py_tensor.cpp"],
+)
+
+load("@pip_deps//:requirements.bzl", "all_requirements")
+
+py_test(
+    name = "test_python_binding",
```

Comment:
Where is it used? When are those tests executed in CI?

Reply:
To be done